README.md

```diff
@@ -45,6 +45,21 @@ stable version as:
 pip install -U sqlalchemy-stubs
 ```
 
+## Development Setup
+
+First, clone the repo and cd into it, like in _Installation_, then:
+```
+git submodule update --init --recursive
+pip install -r external/mypy/test-requirements.txt
+pip install -e external/mypy
+pip install -e .
+```
+
+Then, to run the tests, simply:
+```
+pytest
+```
```

Comment:
Does it actually work like this? I think one needs to  `export PYTHONPATH=external/mypy` before running the tests. Or this is not needed if mypy is installed?

Reply:
Yeah, installing mypy avoids needing to change the `PYTHONPATH`, which has the added benefit of the instructions working across most platforms